news/2024/2024-10-30-osu-world-cup-2024-qualifiers-recap.md

```diff
@@ -0,0 +1,146 @@
+
+The other rule change relates to the mappools, with the mappooling team opting to lift restrictions from banning more than 1 map from each pool slot (e.g. Hidden, Double Time etc.), allowing teams to completely block a mod from being played. The reasoning behind this is that modern mappools ditched the "slot pooling" meta for a more varied pool, with each map showcasing a different skill, so it didn't make sense to keep this rule in place.
+
+Speaking of the mappool, let's check out what the mappooling team cooked up this week, and later on we'll talk to [chiv](https://osu.ppy.sh/users/6701656) and [Dada](https://osu.ppy.sh/users/6701656) about mappooling for Qualifiers at such a high level of play.
+
+## Mappool recap
+
+[**Download the pool here! (132 MB)**](https://osu.ppy.sh/beatmaps/packs/P263)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2281065609&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![Qualifier mappool table](/wiki/shared/news/2024-10-30-osu-world-cup-2024-qualifiers-recap/mappool.jpg)
+
+The Qualifier pool for the osu! World Cup 2024 featured 11 maps, all custom-made specifically for the tournament. While it was a step-up in difficulty from last year, there were teams who still managed to FC some maps with all 4 players, which just shows that the playing field has been improving just as much, if not more, than the increase in challenge.
+
+Some highlights of the pool include [Crystal Illusion](https://osu.ppy.sh/beatmapsets/2267934#osu/4829693) by [milr_](https://osu.ppy.sh/users/4485933) (HR2), a Hard Rock precision map with challenging elements of flow aim and cut streams that was home to one of the most impressive scores in the entirety of the round, being an SS by [decaten](https://osu.ppy.sh/users/5645231) of team USA, and [We Are One (Original Vocal Mix)](https://osu.ppy.sh/beatmapsets/2267993#osu/4829833) by [Aistre](https://osu.ppy.sh/users/4879380) (DT1), a tricky, extremely fast aim map at a high AR on Double Time, which ended up challenging many players through its unusual patterning reminiscent of an older era.
+
+A final, noteworthy highlight is the return of the osu!mania original [Toromaru](https://osu.ppy.sh/beatmapsets/2268007#osu/4829864) through a map done by [Wispy](https://osu.ppy.sh/users/11106929), slotted in the Hidden pool. The tricky rhythms and structure ended up being quite a challenge in osu!standard as well, as the map was one of the most difficult in the pool by the end of the week, as the statistics suggest.
```

Comment:
```suggestion
The osu! World Cup 2024 Qualifier pool featured an 11-map lineup, all tailor-made to accurately gauge the skill levels of the participating teams. While it marked a notable step up in difficulty from last year, some teams still managed to secure full combos with all four players—a testament to the significant improvements across the competitive field.

Highlights from the pool include [Crystal Illusion](https://osu.ppy.sh/beatmapsets/2267934#osu/4829693) by [milr_](https://osu.ppy.sh/users/4485933) (HR2), a Hard Rock precision map packed with challenging flow aim and cut stream patterns. This map set the stage for one of the round's most impressive scores: an SS by ::{ flag=US }:: [decaten](https://osu.ppy.sh/users/5645231), which was the first SS on any Hard Rock map in OWC since the Round of 32 in 2022! Another standout was [We Are One (Original Vocal Mix)](https://osu.ppy.sh/beatmapsets/2267993#osu/4829833) by [Aistre](https://osu.ppy.sh/users/4879380) (DT1), a blazing-fast Double Time aim map with high AR and throwback patterning that tested players with its nostalgic, old-school flair.

Rounding out the highlights is the return of the osu!mania 4K World Cup 2023 original [Deorbit](https://osu.ppy.sh/beatmapsets/2268007#osu/4829864), skilfully reimagined by [Wispy](https://osu.ppy.sh/users/11106929) for the Hidden pool. With its intricate rhythms and complex structure, this map brought a unique challenge to the osu! World Cup as well, standing out as one of the toughest in the pool by week's end, as the stats show.
```